db/sqlc/migrations/000005_actions.up.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE IF NOT EXISTS actions(
+    -- The ID of the action.
+    id INTEGER PRIMARY KEY,
+
+    -- The session ID of the session that this action is associated with.
+    -- This may be null for actions that are not coupled to a session.
+    session_id BIGINT REFERENCES sessions(id) ON DELETE CASCADE,
+
+    -- The account ID of the account that this action is associated with.
+    -- This may be null for actions that are not coupled to an account.
+    account_id BIGINT REFERENCES accounts(id) ON DELETE CASCADE,
+
+    -- An ID derived from the macaroon used to perform the action.
+    macaroon_identifier BLOB,
+
+    -- The name of the entity who performed the action.
+    actor_name TEXT,
+
+    -- The name of the feature that the action is being performed by.
+    feature_name TEXT,
+
+    -- Meta info detailing what caused this action to be executed.
+    trigger TEXT,
```

Comment:
nit: trigger is a reserved keyword in SQL, so could potentially consider using another name for this column.